std_msgs/UInt8 path_id
nav_msgs/Path path
